python/graphstorm/model/rgcn_encoder.py

```diff
@@ -138,7 +138,9 @@ def forward(self, g, inputs):
         else:
             inputs_src = inputs_dst = inputs
 
-        hs = self.conv(g, inputs_src, mod_kwargs=wdict)
+        dst_h = {k: th.zeros((g.num_dst_nodes(k), self.in_feat),
```

Comment:
Why we pass-in an all-zero dst_h?

Reply:
there is an error if i don't pass zeros. i don't remember what the error was.